src/unity/paths/orgs.yml

```
@@ -9,23 +9,23 @@ post:
     content:
       application/json:
         schema:
-          $ref: "../schemas/OrganizationCreateRequest.yml"
+          $ref: '../schemas/OrganizationCreateRequest.yml'
   responses:
-    "201":
+    '201':
       description: Organization created
       content:
         application/json:
           schema:
-            $ref: "../schemas/OrganizationWithToken.yml"
-    "400":
-      description: Invalid request
-      $ref: "../../common/responses/ServerError.yml"
-    "401":
+            $ref: '../schemas/OrganizationWithToken.yml'
+    '401':
       description: Unauthorized bearer token
```

```suggestion
      description: Unauthorized. The token passed doesn't have permissions necessary to complete the request.
```
Is this only `Bearer` scheme?